gramex/handlers/mlhandler.py

```diff
@@ -31,6 +32,14 @@
     'sklearn.neighbors',
     'sklearn.neural_network',
     'sklearn.naive_bayes',
+    'sklearn.decomposition'
+
+]
+SKLEARN_DECOMPOSE_ATTRS = [
+    "components_",
+    "explained_variance_",
+    "explained_variance_ratio_",
+    "singular_values_"
```

Comment:
Could you please check what other attributes other models provide. They may provide more. Or less. Could we expose all model attributes rather than pre-selecting?

Reply:
Yes, there are many classes in `sklearn.decomposition` that have attributes other than these. By convention they all have names that end in `_`. We'll just return any attribute a model has that ends in an underscore.

Reply:
@sanand0  Done, please take a look at the last commit.